stonesoup/regulariser/particle.py

```diff
@@ -24,17 +26,19 @@ class MCMCRegulariser(Regulariser):
     ----------
     .. [1] Robert, Christian P. & Casella, George, Monte Carlo Statistical Methods, Springer, 1999.
 
-    .. [2] Ristic, Branco & Arulampalam, Sanjeev & Gordon, Neil, Beyond the Kalman Filter:
+    .. [2] Ristic, Branko & Arulampalam, Sanjeev & Gordon, Neil, Beyond the Kalman Filter:
         Particle Filters for Target Tracking Applications, Artech House, 2004. """
 
+    transition_model: TransitionModel = Property(doc="Transition model used for prediction")
+
     def regularise(self, prior, posterior, detections):
         """Regularise the particles
```

Comment:
Could you also pass an optional transition model, using the `hypothesis.prediction.transition_model`, from the updater, and falling back to provide one (on `self`) if not provided. Possibly via `**kwargs` or even passing `hypothesis` instead?